src/main/java/ladderGame/model/Players.java

```diff
@@ -0,0 +1,39 @@
+package ladderGame.model;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+
+public class Players {
+    private static final String EXCEPTION_MESSAGE_DUPLICATION_NAME = "참여자들의 이름은 중복될 수 없습니다.";
+    private static final String EXCEPTION_MESSAGE_LESS_THAN_MINIMUM = "참여자의 이름은 두 개 이상이어야 합니다.";
+    private static final int MINIMUM_NAME_COUNT = 2;
+
+    private final List<Player> players;
+
+    public Players(List<String> names) {
+        validate(names);
+
+        players = new ArrayList<>();
```

Comment:
```suggestion
        this.players = new ArrayList<>();
```
이부분은 stream으로 초기화해보면 어떨까요?